src/utils/modules.js

```diff
@@ -330,12 +256,16 @@ export class Modules {
     }
   }
 
-  getRequire(): Value {
-    if (!(this._require instanceof FunctionValue)) {
-      this._require = this._getGlobalProperty("require");
-      if (!(this._require instanceof FunctionValue)) this._require = this._getGlobalProperty("__r");
-    }
-    return this._require;
+  getRequireInfo(): void | RequireInfo {
```

Comment:
Perhaps make this an actual getter?

Reply:
We don't generally use getters in our code base.